Regression_test/CMakeLists.txt

```diff
@@ -22,3 +22,10 @@ add_executable(${PROJECT_NAME} ${SOURCE_FILES} ${HEADER_FILES})
 add_definitions("-DSOFA_SRC_DIR=\"${CMAKE_SOURCE_DIR}\"")
 target_link_libraries(${PROJECT_NAME} PUBLIC Sofa.Testing)
 target_link_libraries(${PROJECT_NAME} PUBLIC Sofa.Component)
+
+sofa_add_targets_to_package(
+    PACKAGE_NAME SofaGui
+    TARGETS ${PROJECT_NAME} AUTO_SET_TARGET_PROPERTIES
+    INCLUDE_SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}"
+    INCLUDE_INSTALL_DIR "SofaGui/${PROJECT_NAME}"
+    )
```

Comment:
Why under SofaGui ?

Reply:
I copied-paste from runSofa 🙊 which has this package name as well... what should we set ?